test/integration/line_resources/libraries/file_ext.rb

```diff
@@ -27,9 +27,6 @@ def to_s
   end
 
   def size_lines
-    if @file.exist?
-      raise "FileExt #{path} is not a file" unless @file.file?
-    end
-    @file.exist? ? @file.content.lines.count : 0
+    @file.content ? @file.content.lines.count : 0
```

Comment:
@file.content.lines would error if the file had no content because @file.content would be nil.  So I want to fix that.  Reading the inspec source seemed to imply that @file.content was defined for  larger variety of files than just file? match.  It seemed right to let inspec return what it wanted for @file.content and just test it for the number of lines.